frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -266,6 +268,13 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
     };
   },
   function hpAction(data): ActionCardProps {
+    // Default content temporarily implemented during COVID-19 Outbreak
+    const covidMessage = <>
+        <span className="subtitle">Due to the Covid-19 health crisis, Housing Courts in New York City are closed. 
+          You can still make the forms to take your landlord to court but you will not be able to file them until the courts re-open.</span>
+        <span className="subtitle">If you are facing an emergency such as lack of heat and/or hot water, call the <strong>Housing Court Answers Hotline at (212) 962-4795</strong> to get assistance Mon-Fri, 9am-5pm. 
```

Comment:
Huh, I'm curious if we want to wrap that phone number in a [`tel:` link](https://css-tricks.com/the-current-state-of-telephone-links/).  That way if a mobile user clicks on it, it should start their phone app, so they don't have to write down the number and then visit their phone app and manually enter it.

Reply:
good idea!